playtest.py

```diff
@@ -1,19 +1,37 @@
+from io import BytesIO
```

Comment:
Could you add a separate test module for your testing?  This file is just a small example of how to connect to play.clickhouse.com

Reply:
fixed tests for servers version started from 19.16.19.85

Reply:
revert back original file